UI/api-interface.cpp

```diff
@@ -370,6 +370,39 @@ struct OBSStudioAPI : obs_frontend_callbacks {
 		return (void *)main->AddDockWidget((QDockWidget *)dock);
 	}
 
+	void obs_frontend_add_extra_browser_dock(const char *title,
+						 const char *url,
+						 const char *uuid) override
+	{
+		QString qstrTitle = QT_UTF8(title);
+		QString qstrURL = QT_UTF8(url);
+		QString qstrUUID = QT_UTF8(uuid);
+
+		return main->AddExtraBrowserDock(qstrTitle, qstrURL, qstrUUID,
+						 true);
+	}
+
+	void obs_frontend_get_extra_browser_docks(
+		std::vector<std::string> &strings) override
+	{
+		for (int i = 0; i < main->extraBrowserDocks.size(); i++) {
+			QAction *action =
+				main->extraBrowserDockActions[i].data();
+			QString url = main->extraBrowserDockTargets[i];
+			QString uuid = action->property("uuid").toString();
+			strings.emplace_back(QT_TO_UTF8(action->text()));
```

Comment:
Instead of using `QT_TO_UTF8()`, the `.toStdString()` function can be used.

Reply:
Done, thanks.